trackpy/tests/test_linking.py

```diff
@@ -701,6 +701,12 @@ def setUp(self):
         self.linker_opts = dict(link_strategy='numba')
 
 
+class TestHybridLink(SubnetNeededTests):
```

Comment:
Just a thought: what about making a mock subnetlinker and testing if the subnetlinker gets the expected subnets (sorted / not sorted, with / without null particle)?  In a separate PR of course